### PR TITLE
Add black to requirements.txt

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ To perform codestyle linting and ansible sanity checks, run `make lint sanity`.
 To (re-)record tests, you first need to setup a pulp instance ([pulplift](https://github.com/pulp/pulplift) is recommended here).
```
Comment (markgoddard) [inline README.md]: Seems lint is mentioned on L58, I just missed it.

**Review by mdellweg — DISMISSED** [review]

**Comment by mdellweg** [comment]
OK, this didn't work out. Maybe split it out in a `lint-requirements.txt` together with flake that you install in the lint workflow.

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ansible_runner<2.0; python_version < '3.6'
 ansible_runner; python_version >= '3.6'
+black
```
Comment (mdellweg) [inline requirements.txt]: What about we restrict this to newer pythons
```suggestion
black; python_version >= '3.6'
```

Comment (markgoddard) [reply requirements.txt]: Nice idea.

**Comment by markgoddard** [comment]
The CI failure looks unrelated. Is there any way to recheck?

**Comment by mdellweg** [comment]
> The CI failure looks unrelated. Is there any way to recheck?

Fixtures service seems to be down. Let's try again. But in the end we can merge this regardless.

**Comment by mdellweg** [comment]
Can you squash the commit into one?

**Comment by mdellweg** [comment]
FTR: `Cannot connect to host fixtures.pulpproject.org:443` the service seems to be down. Merging anyway.

**Pull request merged**